Assets/Scripts/GlobalRunes.cs

```diff
@@ -24,12 +24,17 @@ private struct RuneUpdate
     // Start is called before the first frame update
     void Start()
     {
-        updateQ = new Queue<RuneUpdate>();
+       
     }
 
     // Update is called once per frame
     void Update()
     {
+        if (updateQ == null)
```

Comment:
Why did you move this to the update function? I thought it would have made more sense to initialize it in `Start()` or `Awake()` rather than checking every frame that it's null

Reply:
So for this one, I had a lot of issues with an "Object null reference" which I think was occurring because a Rune script was trying to access updateQ before the Start function of GlobalRunes was called. So to fix this, I made it check if it needed to create the queue before it needed to do something with it. There might be a way to make it work for Start, but this solution was simple and fixed the issue.